packages/react-widgets/src/hooks/useWidgetFilterValues.js

```diff
@@ -8,13 +8,14 @@ import { useMemo } from 'react';
  * @param  {object} props
  * @param  {string} props.dataSource - ID of the source to get the filters from.
  * @param  {string} props.id - ID of the widget that applied the filter.
- * @param  {string} props.column - name of column of this widget.
+ * @param  {string=} props.column - name of column of this widget.
```

Comment:
do. you need this `=`? what does it mean?

Reply:
it's means _optional_ in JSDoc >> https://stackoverflow.com/questions/5873927/how-to-indicate-param-is-optional-using-inline-jsdoc